acknowledge_license_cert.html

```diff
@@ -12,7 +12,9 @@
 <h3>License information</h3>  
 <div class="paragraph">   
 Unless otherwise stated, this code is subject to the terms of the <a href="license.txt">
-GNU General Public License version 3 or higher (GPL)</a>, a copy of which is included here for your convenience.
+GNU General Public License version 3 or higher (GPL)</a>, a copy of which is included here for your convenience.<br>
```

Comment:
see if that works. I am assuming from your post it is called mpl_license.txt instead of license.txt